### PR TITLE
make release notes tool work with more than 30 PRs

### PR DESCRIPTION
**Pull request by joejulian**
**What type of PR is this?**

bug

**What this PR does/ why we need it**:

This will now deal with issue paging in the github api.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by s12chung — APPROVED** [review]

File: tools/cmd/release-notes/release-notes.go [hunk 1]
```diff
@@ -39,28 +39,37 @@ func getMilestone(title string, client *github.Client) (*github.Milestone, error
...
-		return nil, err
-	}
-	result := []*github.Issue{}
-	for _, pullRequest := range pullRequests {
-		if pullRequest.Milestone.GetID() != milestone.GetID() {
-			continue
+	result := map[int64]*github.Issue{}
+	options := github.IssueListByRepoOptions{Milestone: strconv.Itoa(milestone.GetNumber()), State: "closed"}
+	options.Page = 0
+	options.PerPage = 100
```
Comment (dkoshkin) [inline tools/cmd/release-notes/release-notes.go:50]: Is the default 30 results/page and max is 100?

**Review by dkoshkin — APPROVED** [review]

**Comment by samvantran** [comment]
Nice 👍

File: tools/cmd/release-notes/release-notes.go (hunk 1 above)
Comment (joejulian) [reply tools/cmd/release-notes/release-notes.go:50]: correct, though the 30 doesn't seem to be documented.

**Comment by joejulian** [comment]
@mesosphere-mergebot backport  release/3 release/4

**Pull request merged**